data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -32,7 +32,15 @@ The following table outlines the mandatory and optional parameters you will find
+|       Parameter        |                                                                                               Description                                                                                                |                                              Example                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Source                 | Mandatory. Select your source from the drop down menu.                                                                                                                                                   | REST API                                                                                          |
+| HTTP Method            | Mandatory.                                                                                                                                                                                               | This will be either GET or POST.                                                                  |
+| API Response Format    | Mandatory. Use this field to specify a response format of the endpoint. Currently, the Connections UI only supports JSON responses.                                                                      | JSON                                                                                              |
+| Records Root JSONPath  | Mandatory. Specify the JSON path for the results. If the JSON response top-element is an object, use `$.data` to return the parent object of the response, which will contain the array. | `$.data`, `$`                                                                                     |
```

Comment:
Unfortunately, this is still not clear. If you have an understanding of how this works, is it possible to do example pairs where we have the data, the value for **Records Root JSONPath**,  and what a particular value results in?

Same with path to iterate.

Reply:
Ah just saw the later bit. Perhaps an on-page hyperlink to the header below?

Reply:
Get rid of the following and just link to the examples lower down:

> If the JSON response top-element is an object, use `$.data` to return the parent object of the response, which will contain the array.

This is factually incorrect and it's explained better there anyway. You only need $.data if the JSON response's records are in a "data" key

Reply:
I believe all the records we tried that returned are wrapped in a `data` object. Kishan and I looked into it, which is why I thought I had gotten it correct. I understand there might be other payloads then. But I tried on a separate API (Fruit API) and it was also wrapped in data.

Reply:
I believe this should say something along the lines of:

Specify the JSON path for the records in the JSON response. If the JSON response top-most element is an **array**, use `$.data` to treat each element as a record.